doc/sai-ptf/vlan_test_plan.md

```diff
@@ -0,0 +1,480 @@
+# SAI Vlan Test plan
+- [SAI Vlan Test plan](#sai-vlan-test-plan)
+  - [Overriew](#overriew)
+    - [Testbed](#testbed)
```

Comment:
i suggest to have a separate doc to discuss t0 topology since lots of tests (lag/vlan will be sharing the same topology).